select_command/_index.md

```diff
@@ -0,0 +1,60 @@
+
+# <center><b>Mission 1: Save the Planet of Fun From Boredom!</b></center>

```

Comment:
You don't need the center tag or the b tag if you use # (which is equivalent to h1)